### ISSUE TITLE
GetFromHTTPRequest not working

### ISSUE THREAD

**Issue by rosostolato** [capacitor-community/sqlite#534]
**Describe the bug**
The `GetFromHTTPRequest` method is not working on Android when you have a Firebase storage link, which contains query params. I guess it's an issue similar to this one https://github.com/jepiqueau/jeep-sqlite/issues/39.

Error: `GetFromHTTPRequest: length=0; index=-3`

**To Reproduce**
Here is an example of one of my databases stored in my Firebase storage project:
https://firebasestorage.googleapis.com/v0/b/ipmacae-app.appspot.com/o/bibles%2FNVI.zip?alt=media&token=3c035e3e-7a5e-4532-bbf2-4e93e5d69b61

To make `jeepSqlite` work, I need to add a final param called `ext` with the extension of the database. It works just fine on the web but doesn't work on Android and probably on iOS either.

```ts
  async loadDatabaseFromServer(
    database: string,
    url: string,
    ext: 'db' | 'zip',
  ): Promise<void> {
    let databases: string[] = [];
    try {
      const result = await this.sqliteConnection.getDatabaseList();
      databases = result.values ?? [];
    } catch {
      // do nothing
    }
    if (!databases.includes(`${database}SQLite.db`)) {
      await this.sqliteConnection.getFromHTTPRequest(
        `${url}&ext=.${ext}`,
        true,
      );
      const result = await this.sqliteConnection.getDatabaseList();
      databases = result.values ?? [];
    }
    await Toast.show({
      text: `Database ${database} loaded from server. ${databases.values}`,
      duration: 'long',
    });
  }
```

**Expected behavior**
It should toast the success message.

**Screenshots**
![image](https://github.com/capacitor-community/sqlite/assets/27731641/365ddbb8-c546-4f75-895c-6890179d2f11)

**Smartphone (please complete the following information):**

- Device: Pixel 7 emulator
- OS: API 34

**Comment by rosostolato** [comment]
I found what the problem is, firebase returns a disposition with `filename*=utf-8"{fileName}`. I tried to fix it by signing the blob just like mentioned [here](https://stackoverflow.com/questions/52776659/how-to-correctly-set-content-disposition-for-firebase-storage) but it still doesn't work.

If it's too overkill to handle this scenario on the plugin, I will try to move to another storage service, e.g. CloudFlare.
